### PR TITLE
include pokemons in the store after api call

### PR DESCRIPTION
**Pull request by fargito**

### CODE REVIEW

File: frontend/src/redux/Pokemons/reducer.ts [hunk 1]
```diff
@@ -2,9 +2,9 @@ import { ActionType, getType } from 'typesafe-actions';
 
 import { AnyAction } from 'redux';
 import { pokemonType } from './types';
-import { fetchPokemonsSuccess } from './actions';
+import { fetchPokemonsSuccess, fetchPokemonSuccess } from './actions';
 
-export type PokemonsAction = ActionType<typeof fetchPokemonsSuccess>;
+export type PokemonsAction = ActionType<typeof fetchPokemonsSuccess | typeof fetchPokemonSuccess>;
```
Comment (AleBlondin) [inline frontend/src/redux/Pokemons/reducer.ts]: list

File: frontend/src/redux/Pokemons/reducer.ts [hunk 2]
```diff
@@ -15,6 +15,10 @@ const reducer = (state: PokemonsState = initialState, action: AnyAction) => {
   switch (typedAction.type) {
     case getType(fetchPokemonsSuccess):
       return typedAction.payload.pokemons;
+    case getType(fetchPokemonSuccess):
+      const newState = { ...state };
+      newState[typedAction.payload.pokemon.id] = typedAction.payload.pokemon;
+      return newState;
```
Comment (AleBlondin) [inline frontend/src/redux/Pokemons/reducer.ts]: return {...state, [typedAction.payload.pokemon.id]: typedAction.payload.pokemon}

File: frontend/src/redux/Pokemons/__tests__/selectors.test.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    93: { id: 93, name: 'haunter', height: 16, weight: 1 },
+    94: { id: 94, name: 'gengar', height: 15, weight: 405 },
+    95: { id: 95, name: 'onix', height: 88, weight: 2100 },
+    96: { id: 96, name: 'drowzee', height: 10, weight: 324 },
+    97: { id: 97, name: 'hypno', height: 16, weight: 756 },
+  },
+};
+
+const pokemonsList = [
+  { id: 91, name: 'cloyster', height: 15, weight: 1325 },
```
Comment (AleBlondin) [inline frontend/src/redux/Pokemons/__tests__/selectors.test.ts]: values de lodash

File: frontend/src/HOC/with-fetch-poke-api.tsx [hunk 4]
```diff
@@ -6,7 +6,7 @@ import ErrorDisplayer from 'components/ErrorDisplayer';
 import { pokemonType } from 'redux/Pokemons/types';
 
 export interface WithFetchAPIType {
-  fetchPokemonsSuccess: (pokemons: []) => any;
+  fetchPokemonsSuccess: (responseBody: any) => any;
```
[the remark this answers is not in the document]
Comment (fargito) [reply frontend/src/HOC/with-fetch-poke-api.tsx]: Done

File: frontend/src/redux/Pokemons/reducer.ts (hunk 1 above)
Comment (fargito) [reply frontend/src/redux/Pokemons/reducer.ts]: Done

File: frontend/src/redux/Pokemons/reducer.ts (hunk 2 above)
Comment (fargito) [reply frontend/src/redux/Pokemons/reducer.ts]: Done

File: frontend/src/redux/Pokemons/__tests__/selectors.test.ts (hunk 3 above)
Comment (fargito) [reply frontend/src/redux/Pokemons/__tests__/selectors.test.ts]: Done

File: frontend/src/services/Pokemons/PokemonsNormalizer.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+export const normalize = pokemons => {
```
[the remark this answers is not in the document]
Comment (fargito) [reply frontend/src/services/Pokemons/PokemonsNormalizer.js]: Done

**Pull request merged**